phpstan.neon.dist

```diff
@@ -21,6 +21,12 @@ parameters:
         # allow partial parsing of language fragments.
         - "~Variable method call on GraphQL\\\\Language\\\\Parser\\.~"
 
+        # We utilize lazy initialization of non-nullable properties
+        - "~Property .* in isset\\(\\) is not nullable~"
```

Comment:
```suggestion
        - '~Property .+? in isset\(\) is not nullable~'
```

Reply:
Single quotes reducing the amount of backslashes is nice, but double quotes are used in the rest of the file and in the generated baseline.